en-US/skins/README.md

```diff
@@ -10,13 +10,58 @@ Skins are one of the many great features of Quaver. It allows players to complet
 
+# Skinning FAQs
+
 ## How do I create a skin?
 
 Skinning has a very forced organization system and needs to be done correctly in order to work properly. The following list are guides to skin each part of the game.
 
-To get started, it'll be helpful to start with a base skin. You can download both our default bar and arrow skins [here](https://www.dropbox.com/s/09xjrt71rakztnw/Quaver%20Skins.zip?dl=1).
+To get started, it'll be helpful to start with a base skin. You can download both our default bar and arrow skins [here](https://steamcommunity.com/id/janko5/myworkshopfiles/?appid=980610).
 
 * [Gameplay Interface](/docs/Skins/Interface)
 * [4K and 7K](/docs/Skins/Keys)
 * [Sound Effects (SFX)](/docs/Skins/SFX)
 * [Skin.ini](/docs/Skins/Skin.ini)
+
+If you want to start creating skin elements from the scratch, you will need an image editing software of your choice. We made a list of the software below:
+
+   - [Adobe Photoshop](https://www.adobe.com/products/photoshop.html?promoid=PC1PQQ5T&mv=other)
+   - [GIMP](https://www.gimp.org)
+   - [paint.net](https://www.getpaint.net/index.html)
+   - [Figma](https://www.figma.com/login)
+   - [Photopea](https://www.photopea.com)
+
+
+## Where can I get skins that I saw someone played with in Quaver?
+
+There are two ways of getting skins made by other people:
+
+* [Quaver's Steam Workshop](https://steamcommunity.com/app/980610/workshop/)
+  
+  You can get skins in the Steam Workshop by clicking on a skin of your choice, and click Subscribe. After subscribing, Steam will download the skin files from the workshop. When the download is complete, the skin should be available in Skin Settings. Skins that you have currently subscribed to are in blue font color.
+
+* Asking the person for the skin they're using.
+  
+  Some people do not upload their skins in the Steam Workshop. You may ask them for the skin but they may not give it to you.
+
+## Can I make my own edition of someone else's skin?
+Yes, you can make your own edition or use skin elements from other skins and you are allowed to upload them in the Steam Workshop.
+
+However, we suggest giving credit to the original author of the skin if you would use their skin elements.
+
+
+## How do I reload my skin in Quaver?
+With the skin of your choice, reloading the skin is by pressing CTRL + S in the song selection menu or the main menu.
+
+
+# Uploading a Skin to the Workshop
+
+If you want to upload your skin to the workshop, you must abide by [the rules and guidelines](https://support.steampowered.com/kb_article.php?ref=4045-USHJ-3810). Otherwise, your skin will get reported and taken down or will get blocked by the Steam Community Moderators from uploading your skins in the workshop.
```

Comment:
```suggestion
If you would like to upload your skin to the workshop, you must abide by [the rules and guidelines](https://support.steampowered.com/kb_article.php?ref=4045-USHJ-3810). Otherwise, you may not be allowed to upload it.
```